### PR TITLE
Certification screen (EXPOSUREAPP-7433)

### PR DESCRIPTION
**Pull request by jurajkusnier**
Main Cerification screen with some cards. Cards will be adjusted in the following PR.

device-2021-05-31-180554

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/green_certificate_strings.xml [hunk 1]
```diff
@@ -26,4 +26,16 @@
     <string name="request_gc_dialog_positive_button">OK</string>
     <!-- XBUT: Request green certificate exit dialog negative button -->
     <string name="request_gc_dialog_negative_button">Abbrechen</string>
+    <!-- XTXT: Green certificate main screen title -->
+    <string name="certification_screen_title">Nachweise</string>
+    <!-- XTXT: Green certificate menu item -->
+    <string name="menu_certification_information">Information</string>
+    <!-- XTXT: Green certificate main screen header text -->
+    <string name="green_certification_header_info">Hier werden ihre digitalen Impfzertifikate und Testzertifikate angezeigt.</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values-de/green_certificate_strings.xml:34]: please spell "**Ihre** Impfzertifikate" with a capital I.

File: Corona-Warn-App/src/main/res/values/green_certificate_strings.xml [hunk 2]
```diff
@@ -26,4 +26,17 @@
     <string name="request_gc_dialog_positive_button">OK</string>
     <!-- XBUT: Request green certificate exit dialog negative button -->
     <string name="request_gc_dialog_negative_button">Abbrechen</string>
+    <!-- XTXT: Green certificate main screen title -->
+    <string name="certification_screen_title">Nachweise</string>
+    <!-- XTXT: Green certificate menu item -->
+    <string name="menu_certification_information">Information</string>
+    <!-- XTXT: Green certificate main screen header text -->
+    <string name="green_certification_header_info">Hier werden ihre digitalen Impfzertifikate und Testzertifikate angezeigt.</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values/green_certificate_strings.xml:34]: same here, "Ihre" with capital I

File: Corona-Warn-App/src/main/res/values-de/green_certificate_strings.xml (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values-de/green_certificate_strings.xml]: updated

File: Corona-Warn-App/src/main/res/values/green_certificate_strings.xml (hunk 2 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values/green_certificate_strings.xml]: updated

**Review by ChristinaTsigganos — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/CertificatesViewModel.kt [hunk 3]
```diff
@@ -1,11 +1,77 @@
...
-class CertificatesViewModel @AssistedInject constructor() : CWAViewModel() {
+class CertificatesViewModel @AssistedInject constructor(
+    vaccinationRepository: VaccinationRepository,
+    private val vaccinationSettings: VaccinationSettings
+) : CWAViewModel() {
+
+    val events = SingleLiveEvent<CertificatesFragmentEvents>()
+
+    // TODO: cards should be adjusted in the following PR
+    val screenItems: LiveData<List<CertificatesItem>> =
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/CertificatesViewModel.kt:29]: conside to move some code in functions for better readability of the code...

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    override val viewBinding = lazy {
+        VaccinationBottomInfoCardBinding.inflate(layoutInflater, itemView.findViewById(R.id.container_layout), true)
+    }
+
+    override val onBindData: VaccinationBottomInfoCardBinding.(
+        item: Item,
+        payloads: List<Any>
+    ) -> Unit = { _, _ -> }
+
+    object Item : CertificatesItem, HasPayloadDiffer {
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt:25]: What is the idea behind this?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/CertificatesViewModel.kt (hunk 3 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/CertificatesViewModel.kt:29]: I'll consider it in the following PR focused on cards.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt (hunk 4 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt]: `BottomInfoVaccinationCard` displays information about the digital covid-19 certificate when no certificate is registered. I'm reusing cards from the home screen so using the same implementation with adapter and item view holder. And if you actually talk about `HasPayloadDiffer` for an object, that's leftover from the previous implementation.
It should be updated now

**Review by AlexanderAlferov — APPROVED** [review]
As a "preparation" pr - LGTM 🚀 
Tested on Samsung S8

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt (hunk 4 above)
Comment (BMItter) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt]: all right

**Review by BMItr — APPROVED** [review]
lgtm

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Pull request merged**